app/shared/models.py

```diff
@@ -2,10 +2,10 @@
 
 from django.db import models
 
+
 class CommonInfo(models.Model):
```

Comment:
@andela-kanyanwu IMO having a `CommonInfo` model that holds ALL the possible common fields might not be the best. A more robust approach could be to break this common behaviors into a set of resuable mixin models such that each one mixes in a only that specific kind of behaviour into the model it is added to. In this case, we could have a `Timestampable` or `TimestampMixin` model whose sole purpose is to add timestamp-related fields like `date_created` and `date_modified` to other models (where needed). We could create other reusable models for other behaviours when needed. You can read [Django Model Behavious - the reusable mixins section](http://blog.kevinastone.com/django-model-behaviors.html) for a better explanation on this pattern.
Also note that these reusable mixin models will not be the sole base class for the inheriting models when used. The child models will still inherit form `models.Model` but will have the reusable behaviour as a n added mixin.